public/app/features/explore/Logs.tsx

```diff
@@ -404,14 +406,21 @@ class UnthemedLogs extends PureComponent<Props, State> {
 
     const scanText = scanRange ? `Scanning ${rangeUtil.describeTimeRange(scanRange)}` : 'Scanning...';
 
+    const contextQuery = (logsQueries ?? []).find(
+      (q) =>
+        q.refId === contextRow?.dataFrame.refId &&
+        q.datasource != null &&
+        q.datasource.type === contextRow.datasourceType
+    );
```

Comment:
It looks like we are using the same logic at couple of different places, so maybe we could create a util function out of this:
<img width="962" alt="image" src="https://user-images.githubusercontent.com/30407135/233017672-bcdedab8-dd86-4047-9dbe-476d3f8e0b37.png">

Reply:
Fixed in [ae74f25](https://github.com/grafana/grafana/pull/66777/commits/ae74f25435b784104fbd9283e1c53863f9ecf761)